pkgs/games/eduke32/default.nix

```
@@ -112,48 +124,71 @@ in stdenv.mkDerivation (finalAttrs: {
       genericName = "Shadow Warrior source port";
       categories = [ "Game" ];
     })
+    (makeDesktopItem {
+      name = "fury";
+      icon = "fury";
+      exec = "${furyWrapper}";
+      comment = "Ion Fury eduke32 source port";
+      desktopName = "Ion Fury";
+      genericName = "Ion Fury source port";
+      categories = [ "Game" ];
+    })
   ];
 
   enableParallelBuilding = true;
 
-  installPhase = ''
-    runHook preInstall
-
-    install -Dm755 -t $out/bin eduke32 mapster32 voidsw wangulator
-  '' + lib.optionalString stdenv.isLinux ''
-    makeWrapper $out/bin/eduke32 $out/bin/${wrapper} \
-      --set-default EDUKE32_DATA_DIR /var/lib/games/eduke32 \
-      --add-flags '-g "$EDUKE32_DATA_DIR/DUKE3D.GRP"'
-    makeWrapper $out/bin/voidsw $out/bin/${swWrapper} \
-      --set-default EDUKE32_DATA_DIR /var/lib/games/eduke32 \
-      --add-flags '-g"$EDUKE32_DATA_DIR/SW.GRP"'
-    mkdir -p $out/share/icons/hicolor/scalable/apps
-    gm convert "./source/duke3d/rsrc/game_icon.ico[10]" $out/share/icons/hicolor/scalable/apps/eduke32.png
-    install -Dm644 ./source/sw/rsrc/game_icon.svg $out/share/icons/hicolor/scalable/apps/voidsw.svg
-  '' + lib.optionalString stdenv.isDarwin ''
-    mkdir -p $out/Applications/EDuke32.app/Contents/MacOS
-    mkdir -p $out/Applications/Mapster32.app/Contents/MacOS
-    mkdir -p $out/Applications/VoidSW.app/Contents/MacOS
-    mkdir -p $out/Applications/Wangulator.app/Contents/MacOS
-
-    cp -r platform/Apple/bundles/EDuke32.app/* $out/Applications/EDuke32.app/
-    cp -r platform/Apple/bundles/Mapster32.app/* $out/Applications/Mapster32.app/
-    cp -r platform/Apple/bundles/VoidSW.app/* $out/Applications/VoidSW.app/
-    cp -r platform/Apple/bundles/Wangulator.app/* $out/Applications/Wangulator.app/
-
-    ln -sf $out/bin/eduke32 $out/Applications/EDuke32.app/Contents/MacOS/eduke32
-    ln -sf $out/bin/mapster32 $out/Applications/Mapster32.app/Contents/MacOS/mapster32
-    ln -sf $out/bin/voidsw $out/Applications/VoidSW.app/Contents/MacOS/voidsw
-    ln -sf $out/bin/wangulator $out/Applications/Wangulator.app/Contents/MacOS/wangulator
-  '' + ''
-    runHook postInstall
-  '';
+  installPhase =
+    ''
+      runHook preInstall
+
+      install -Dm755 -t $out/bin eduke32 mapster32 voidsw wangulator
+    ''
+    + lib.optionalString stdenv.isLinux ''
+      makeWrapper $out/bin/eduke32 $out/bin/${wrapper} \
+        --set-default EDUKE32_DATA_DIR /var/lib/games/eduke32 \
+        --add-flags '-j "$EDUKE32_DATA_DIR"' \
+        --add-flags '-gamegrp DUKE3D.GRP'
+      makeWrapper $out/bin/voidsw $out/bin/${swWrapper} \
+        --set-default EDUKE32_DATA_DIR /var/lib/games/eduke32 \
+        --add-flags '-j"$EDUKE32_DATA_DIR"' \
+        --add-flags '-gSW.GRP'
```

```suggestion
        --add-flags '-j "$EDUKE32_DATA_DIR"' \
        --add-flags '-g SW.GRP'
```

Style inconsistency compared to other wrappers

---

`voidsw` requires `-j` without the space, but `eduke32` also works without the space, so I made those consistent.

Also `voidsw` only looks for SW.GRP so I removed that line.